_posts/2024-08-23-GSoC-CSL.md

```diff
@@ -0,0 +1,65 @@
+
+For several years, JabRef users have been requesting support for CSL in its LibreOffice integration. Tackling this problem statement this summer as a part of my project, I am excited to announce that this highly anticipated feature is now available!
+
+## What's New?
+
+With this integration, JabRef users can now:
+
+1. **Choose a CSL Style**: Select from a vast array of CSL styles to match the requirements of your academic work.
+2. **Cite with Flexibility**: Insert citations as well as in-text citations (similar to LaTeX's `\cite` and `\citet` commands) directly into your LibreOffice document.
+3. **Auto-generate Bibliographies**: Watch as your bibliography automatically populates based on your citations, with just a single click.
+4. **Update Bibliographies**: Easily refresh your bibliography as you add new citations to your document.
+
+## Getting Started
+
+To start using this new feature:
+
+1. Download the [development version of JabRef](https://builds.jabref.org/main/)
+2. Connect to a running LibreOffice document instance from the [Libre/OpenOffice Panel](https://docs.jabref.org/cite/openofficeintegration)
+3. Click on "Select style" from the panel
+4. Choose your preferred CSL style from the list and click OK
+5. Select an entry (or a group of entries) and click "Cite" to insert citations into the document
+6. Click on the "Make/Sync bibliography" on the panel to auto-generate a bibliography section in your document based on your citations and the selected CSL style
+
+### Demo
+
+#### Selecting a style
+
+![Selecting a style](../img/selecting-a-style.gif)
+
+#### Inserting citations
+
+![Inserting citations](../img/cite.gif)
+
+#### Generating bibliography
+
+![Generating bibliography](../img/bibliography.gif)
+
+#### Updating bibliography
+
+![Updating bibliography](../img/refresh.gif)
+
+## Summary
+
+This CSL integration is a direct result of working on valuable user feedback, addressing long-standing feature requests ([#119](https://github.com/JabRef/jabref/issues/119) and [#2146](https://github.com/JabRef/jabref/issues/2146) on our GitHub).
+
+If you are interested in the technical details of how this project was undertaken, do check out the [wiki](https://github.com/JabRef/jabref/wiki/GSoC-2024-%E2%80%90-Improved-CSL-Support-%28and-more-LibreOffice%E2%80%90JabRef-integration-enhancements%29).
+
+I hope this new feature enhances your research and writing process. As always, [we welcome your feedback and suggestions for further improvements](https://discourse.jabref.org/c/feedback/3).
```

Comment:
Use passive voice in second part. "We" sounds strange, b/c all other parts are "I". Since you also welcome feedback *too* I suggest passive voice